src/k8s-engine/lima.ts

```diff
@@ -712,7 +727,10 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     return Math.random().toString().substring(2, desiredLength + 2);
   }
 
-  protected async showSudoReason(explanations: Array<string>): Promise<void> {
+  /**
+   * Show the dialog box describing why sudo is required.
+   */
+  protected async showSudoReason(this: unknown, explanations: Array<string>): Promise<void> {
```

Comment:
Same thing: where does the `this` parameter come from, and why isn't its type known?

Reply:
(See the other comment on why the `this` parameter exists at all.)

Here, I'm using `unknown` so that TypeScript has zero information about the receiver, so that we can't access it at all.  This basically makes it a static method (that is, it can't access `this` at all) while still using the more ergonomic syntax.